gradle/project.gradle.kts

```diff
@@ -53,7 +51,8 @@ kotlin {
 }
 
 allprojects {
-    buildDir = file(properties["player.buildDir"] ?: "build").resolve(name)
+    apply(plugin = "kotlin")
```

Comment:
The plugin is already required by the root project, but the subprojects can't use it unless you apply it there too. One could also move this to `widgets.gradle.kts`, but since everything in this application uses kotlin, I like to configure it centrally.

Reply:
ok